src/Cli/PhpCommands.php

```diff
@@ -131,66 +149,135 @@ public function phpRpmUpdate()
 
         $datecode = date("Ymd");
 
+        // TODO: convert this snippet of bash from the old script to php.
+        // Purpose: finds the current php_version lines from all php.spec files.
         exec("grep '^%define php_version ' $work_dir/php*/php.spec | sed -e 's#[^ ]* *php_version *##'", $versions, $status);
 
         $updated_versions = [];
         foreach ($versions as $version) {
+            $next_version = $this->nextVersionThatExists($version);
 
-          $next_version = $this->next_version_that_exists($version);
+            if ($next_version != $version) {
+                $this->say("$next_version is available, but we are still on version $version");
 
-          if ($next_version != $version) {
-            $this->say("$next_version is available, but we are still on version $version");
+                // TODO: we need to determine if there is already an open pull request that contains $next_version
+                $this->updateSpec($next_version, $datecode, $work_dir);
+                $updated_versions[] = $next_version;
+            } else {
+                $this->say("$version is the most recent version");
+            }
+        }
 
-            // TODO: we need to determine if there is already an open pull request that contains $next_version
-            $this->update_spec($next_version, $datecode, $work_dir);
-            $updated_versions[] = $next_version;
-          }
-          else {
-            $this->say("$version is the most recent version");
-          }
+        if (empty($updated_versions)) {
+            $this->logger->notice("Nothing was updated.");
+            return;
         }
 
-        if (!empty($updated_versions)) {
-            $all_updated_versions = $this->pretty_implode(', ', ' and ', $updated_versions);
-            $branch = 'php-' . implode('-', $updated_versions);
-            $message = "Update to PHP $all_updated_versions";
-            $rpmbuild_php
-                ->createBranch($branch)
-                ->add('php-*')
-                ->commit($message)
-                ->push('origin', $branch)
-                ->pr($message);
+        $all_updated_versions = $this->prettyImplode(', ', ' and ', array_map(function ($v) {
+            return "php-$v";
+        }, $updated_versions));
+        $preamble = $this->preamble();
+        $message = "{$preamble}{$all_updated_versions}";
+        $this->logger->notice("Commit message {message}", ['message' => $message]);
+
+        // Create a set of version ids from the commit message
+        $vids = new VersionIdentifiers();
```

Comment:
Same as the other command: set values that define whether existing PRs are matching or not